features/LAY000_Layers-information.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@LAY
+@version1
+@E00020
+
+Feature: LAY000 - Layers Information
+    The rule verifies the presence of IFC entities used to assign layers (also known as, CAD layer) to collection of elements. 
+    This is used mainly for grouping and visibility control, and in general to organise geometry into groups that may be shown or hidden.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Geometry_Layer/content.html
+
+
+    Scenario: Check for activation
+
+    Given an IfcProduct
+    Given its attribute Representation
+    Given its attribute Representations
+    Given its attribute LayerAssignments
+
+    Then The IFC model contains information on geometry layers
```

Comment:
geometry layers != layers information

Shouldn't this be consistent?

Reply:
[Changed it ](https://github.com/buildingSMART/ifc-gherkin-rules/pull/256/commits/9f838a0518dfea291bdec4fb3477ce8ae05665af)to 
`The IFC model contains information on the selected functional part`
See https://github.com/buildingSMART/ifc-gherkin-rules/pull/257#discussion_r1712626433